cpp/src/rolling/range_window_bounds.cpp

```diff
@@ -45,7 +46,9 @@ struct range_scalar_constructor {
       static_cast<duration_scalar<T> const&>(range_scalar_));
   }
 
-  template <typename T, CUDF_ENABLE_IF(std::is_integral_v<T> && not cudf::is_boolean<T>())>
+  template <typename T,
+            CUDF_ENABLE_IF(std::is_floating_point_v<T> ||
+                           (std::is_integral_v<T> && not cudf::is_boolean<T>()))>
```

Comment:
```suggestion
  template <typename T,
            CUDF_ENABLE_IF(cudf::is_numeric<T> && not cudf::is_boolean<T>())>
```
This has a somewhat nice symmetry to it -- `if T then static_cast<T_scalar>`

Reply:
Sorry, that should have been `cudf::is_numeric<T>()`, not `cudf::is_numeric<T>`. Will fix shortly.

Reply:
Fixed.